tests/ui_tests/test_numbers.py

```diff
@@ -56,8 +56,8 @@ def test_number_in_range_values(
         assert config.save_button_status()
 
 
-def test_float_in_integer_field(sdk_client_fs: ADCMClient, app_fs, login_to_adcm):
-    _ = login_to_adcm
+def test_float_in_integer_field(sdk_client_fs: ADCMClient, app_fs, login_to_adcm_over_api):
+    _ = login_to_adcm_over_api
```

Comment:
![](https://i.kym-cdn.com/photos/images/newsfeed/000/173/576/Wat8.jpg?1315930535)

Reply:
Who knows... Looks like pylint avoider, should be removed